docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  db:
+    image: mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
you made sure this is the version the staging and prod DB uses? You probably did, but just a reminder in case.

Reply:
I checked with Evan, and we use SQL Server 2014. However, he said it should be all good, and that using 2017 as the tester instance is all good. There are no reliable docker images for pre 2017.

Reply:
Rename this file to `docker-compose.test.yml` so it's clear that it's for testing at first glance.